nemo/collections/nlp/data/language_modeling/megatron/gpt_sft_chat_dataset.py

```diff
@@ -26,21 +26,34 @@
 END_SIGNAL = "\n"
 END_NAME_SIGNAL = "\n"
 
+SCALE = 9
```

Comment:
Can we avoid making this a global variable?

Reply:
It makes it hard to experimentat with different kinds of value models.

Reply:
removed this. only use the value string